urllib3/connectionpool.py

```diff
@@ -381,8 +382,22 @@ def _make_request(self, conn, method, url, timeout=_Default,
         log.debug("\"%s %s %s\" %s %s" % (method, url, http_version,
                                           httplib_response.status,
                                           httplib_response.length))
+
+        header_parsing_errors = extract_parsing_errors(httplib_response.msg)
+        if header_parsing_errors:  # Platform-specific: Implementation.
+            logging.warning(
+                'Errors while parsing headers for %s %s: %s. Unparsed data: %s' %
```

Comment:
I mean rather than formatting the error in the warning string, would be better to just have it formatted by the error itself and the warning string be mostly-static (just add the url for context).